src/nyctaxi/data_processor.py

```
@@ -51,6 +55,38 @@ def preprocess_data(self):
+    
+    
+    def save_to_catalog(self, train_set: pd.DataFrame, test_set: pd.DataFrame, spark):
+        """Save the train and test sets into Databricks tables.
+        
+        Args:
+            train_set: training data
+            test_set: test data
+        """
+
+        train_set_with_timestamp = spark.createDataFrame(train_set).withColumn(
+            "update_timestamp_utc", F.to_utc_timestamp(F.current_timestamp(), "UTC"))   
```

Ruff

Trailing whitespace

Remove trailing whitespace

(W291)